packages/@react-aria/virtualizer/src/Virtualizer.tsx

```diff
@@ -145,6 +145,14 @@ export function useVirtualizer<T extends object, V, W>(props: VirtualizerOptions
     lastFocusedKey.current = focusedKey;
   }, [focusedKey, virtualizer.visibleRect.height, virtualizer, lastFocusedKey, scrollToItem]);
 
+  // Tell the virtualizer to persist the focusedKey and prevent it from being reused.
+  // This is to provent the FocusRing from setting focused to the ListView when a 
+  // focused ListViewItem is scrolled out of the Virtualizer visible rectangle.
+  useMemo(() => {
+    let focusedKeySet = new Set([focusedKey]);
+    virtualizer.persistedKeys = focusedKeySet;
+  }, [focusedKey]);
```

Comment:
useMemo is meant to return a value, you're using it like an effect right now.

```suggestion
  let fofucsedKeySet = useMemo(() => {
    return new Set([focusedKey]);
  }, [focusedKey]);
  
 virtualizer.persistedKeys = focusedKeySet;
```